netci.groovy

```diff
@@ -470,6 +414,66 @@ branchList.each { branchName ->
     }
 }
 
+// adds steps to a job to download coreclr artifacts and corefx artifacts and invoke run-test.sh.
```

Comment:
I personally would prefer such runs in the coreclr repo itself. I don't like the cross repo dependencies being this coupled and I would prefer corefx never have to build coreclr during our CI.